demo/gpu-test7.yaml

```diff
@@ -0,0 +1,97 @@
+# One Namespace with admin access label
+# Two pods, one container each
+# Each container requesting 1 distinct GPU with admin access
+# This demo shows host hardware information exposed via environment variables
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: gpu-test7
+  labels:
+    resource.k8s.io/admin-access: "true"
```

Comment:
Would you suggest that new users set both of these labels in anticipation of upgrading to 1.34? If so, I think we should do that here too.